www/vorj00/ukol_05/js/main.js

```diff
@@ -0,0 +1,180 @@
+const pi = Math.PI;
+const thisYear = new Date().getFullYear();
+
+/* HOMEWORK NUMERO CINQ */
+/**
+ * Pepe's age. Vypište na konzoli smysluplnou oznamovací větu ohledně věku Pepy, pokud znáte jeho rok narození. 
+ * Použijte proměnné a pro výpis použijte zřetězení stringů. Jako názvy proměnných používejte anglické pojmy.
+ */
+const pepe = {};
+pepe.birthYear = 2000;
+pepe.age = thisYear - pepe.birthYear;
+
+console.log('Pepovi je', pepe.age, 'let');
+
+/**
+ * WTF (wow, that's fun). Vypište teplotu v Fahrenheiht, pokud znáte teplotu v Celsius, a také naopak. 
+ * Formát výpisu je: 20°C =  68°F resp. 68°F = 20°C. Výpočet probíhá takto:
+ *     z C na F: vynásobit devíti, vydělit pěti a přičíst 32. 
+ *     z F na C: odečíst 32, vynásobit pěti a vydělit devítkou. 
+ */
+var fahrenheiht = 451;
+var celsius = 20;
+
+var fToC = (fahrenheiht - 32) * 5 / 9;
+var cToF = celsius * 9 / 5 + 32;
+
+console.log(celsius+'°C =',Math.round(cToF)+'°C');
+console.log(fahrenheiht+'°F =',Math.round(fToC)+'°C');
+
+/**
+ * Funkce function fonction funktio. Vemte předchozí úlohy a udělejte z nich funkce. Tj. vytvořte funkce, 
+ * které přijímají argumenty, a na základě argumentů po zavolání vypíše výsledek na konzoli. 
+ * Párkrát zavolejte tyto funkce s různými argumenty. V konzoli také vyzkoušejte, zda fungují vaše funkce. 
+ */
+/* ES6! protože podpora je už (docela) dost dobrá — https://caniuse.com/#search=ES6 */
+getAge = birthYear => {
+    thisYear < birthYear ? console.log("Nemůže se narodit v budoucnu") : console.log('Věk:', thisYear - birthYear);
+};
+
+function temperatureConverter(n, resultUnit){
```

Comment:
tady `n` je hodnota kterou mam a `resultUnit` je jednotka kterou chci? trochu me to matlo

Reply:
jj